### DISCUSSION TITLE
Enable the rust module, but codelldb doesn't auto install.

### DISCUSSION THREAD

**Discussion by 2dogchan** [LazyVim/LazyVim#1043] [General]
By the way, the warning icon had cover the breakingpoint icon, how could I figure out it?
<img width="698" alt="image" src="https://github.com/LazyVim/LazyVim/assets/94974214/63630257-db1b-4c38-9999-a6cc4941fcbc">

**Comment by dpetka2001** [comment]
That's because `Mason` is lazy loaded. So it can't auto install anything until you invoke it. So a simple `:Mason` or the corresponding keymap to invoke it should start auto installing the necessary `ensure_installed`.

**Reply by 2dogchan** [reply]
I had manualy remove the codelldb, and then try start nvim with `Mason` dashboard, but `codelldb` can not be installed.
This is my nvim config.
`https://github.com/2dogchan/lazy-nvim-ide`

**Reply by folke** [reply]
You're overriding ensure_installed here https://github.com/2dogchan/lazy-nvim-ide/blob/490ea2b8061ba81e6be3a8c45d92a2e2f05e71f0/lua/plugins/lsp.lua#L53

You need to use `vim.list_extend` instead. Check the lazyvim code for some examples

**Reply by 2dogchan** [reply]
It works, thanks so much.
